physlr/physlr.py

```diff
@@ -1689,21 +1720,36 @@ def physlr_molecules(self):
             exit_message = "Error: physlr molecule: wrong input parameter(s) " + \
                       "--separation-strategy: " + str(set(alg_list) - alg_white_list)
             sys.exit(exit_message)
+        junctions = []
+        if len(self.args.FILES) > 1:
+            with open(self.args.FILES[1]) as fin:
+                for line in fin:
+                    junctions.append(line.split()[0])
+            print(
+                int(timeit.default_timer() - t0),
+                "Separating junction resulting barcodes into molecules"
```

Comment:
```suggestion
                "Separating junction (contributing / causing) barcodes into molecules"
```
(contributing / causing)  I think sounds better than resulting

Reply:
should I join them with a dash? like junction-causing?

Reply:
Yeah I'd prefer `junction-causing`